### PR TITLE
webrtc: clean up stale bug entry

### PR DESCRIPTION
**Pull request by fippo**
after the only failing test was also removed in
  https://chromiumdash.appspot.com/commit/22122fb6f485dbd5e4e8fe98a88f1c86a234e0e5

### CODE REVIEW

**Comment by fippo** [comment]
hrm... I can't add labels as advised in the template?

🙇 to @sideshowbarker for explaining how this works in chromium slack!
@henbos FYI, please spread internally. We should give those bugs a hard look ;-)

**Comment by sideshowbarker** [comment]
> hrm... I can't add labels as advised in the template?

Dunno — there might be some perms limitation that needs looking at.

But the easier way that _should_ let you add labels is to use the **Triage Mode** thing:

1. Go to https://wpt.fyi/results/webrtc (or whatever other path).
2. At the top of the page, press the Sign in With GitHub button.
3. After you’re logged in that way, toggle the Triage Mode slider on.
4. Drag-select the text of the name of one or more particular subdirectories or .html test paths.
5. In the popup that appears, press the TRIAGE button.
6. In the next popup that appears, paste in a bug URL (and optionally, a label).
7. Press the Triage button in that popup.

…and that’s it. That will create a new PR in the https://github.com/web-platform-tests/wpt-metadata repo, and as soon as all the checks for that PR have run successfully, the PR will get automatically merged into the main branch there — and then a new bug link for what you created will automatically show up in near real time at the wpt.fyi at the place where you added it.

For all that work, you need to be in the GitHub wpt-org team of people who have push/merge access to the set of wpt-org repos — but if you already have that access to the https://github.com/web-platform-tests/wpt repo itself (for reviewing and merging WPT test PRs), then you already have the same access for this wpt-metadata repo.

**Comment by sideshowbarker** [comment]
> hrm... I can't add labels as advised in the template?

oh but I also just now realized that you generally don’t actually want to add the GitHub **do not merge yet** label advised in the template — instead, it’s better to just use the **Triage Mode** feature at the wpt.fyi site itself, and not manually add any GitHub label to the ensuing PR that gets generated, but instead just let the PR get auto-merged after the checks run

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
@fippo I think you should be able to self-merge this now — but if not, lemme know, and I’ll press the button myself

**Comment by fippo** [comment]
No merge button... it seems I am a WPT member but not a collaborator so lack write permissions? Which probably explains why I can't add labels.

If someone could drop me an invite, I promise more clean-up in return ;-)

**Comment by sideshowbarker** [comment]
> No merge button... it seems I am a WPT member but not a collaborator so lack write permissions? Which probably explains why I can't add labels.
> 
> If someone could drop me an invite, I promise more clean-up in return ;-)

Jut now pressed some buttons, and you should be good to go

**Pull request merged**
